IdentityCore/src/configuration/MSIDNetworkConfiguration.m

```diff
@@ -22,29 +22,46 @@
 // THE SOFTWARE.
 
 #import "MSIDNetworkConfiguration.h"
+#import "MSIDURLSessionManager.h"
 
-static NSTimeInterval s_timeout = 30;
-static NSInteger s_retryCount = 2;
+NSTimeInterval MSID_DEFAULT_TIMEOUT_FOR_RESOURCE;
+NSTimeInterval MSID_DEFAULT_TIMEOUT_FOR_REQUEST;
+
+static NSInteger s_retryCount = 1;
+static NSTimeInterval s_retryInterval = 0.5;
 
 @implementation MSIDNetworkConfiguration
 
-+ (void)setTimeout:(NSTimeInterval)timeout
++ (void)load
+{
+    __auto_type defaultConfig = [NSURLSessionConfiguration defaultSessionConfiguration];
```

Comment:
Do we need to create default session configuration on load? I assume we could check those values at a later point.

Reply:
Nope, and also removing MSID_DEFAULT_TIMEOUT_FOR_RESOURCE and MSID_DEFAULT_TIMEOUT_FOR_REQUEST